.github/workflows/lint.yaml

```diff
@@ -40,7 +40,7 @@ jobs:
             | docker run -i --rm ghcr.io/yannh/kubeconform:latest \
             --strict --summary \
             --schema-location default \
-            --schema-location 'https://raw.githubusercontent.com/datreeio/CRDs-catalog/main/{{.Group}}/{{.ResourceKind}}_{{.ResourceAPIVersion}}.json' \
+            --schema-location 'https://raw.githubusercontent.com/stackhpc/CRDs-catalog/refs/heads/update-clusterapi-crds/{{.Group}}/{{.ResourceKind}}_{{.ResourceAPIVersion}}.json' \
```

Comment:
Can we revert this now that https://github.com/datreeio/CRDs-catalog/pull/670 is merged?

Reply:
Yes - done